python/cudf/tests/test_replace.py

```diff
@@ -37,6 +37,55 @@ def test_series_replace():
     sr8 = sr1.replace(sr1[:3], 5)
     np.testing.assert_equal(sr8.to_array(), a8)
 
+    # large input containing null
+    sr9 = Series(list(range(4000000)) + [None])
```

Comment:
Does the input need to be this big to effectively test? This will increase the memory pressure and time it takes to run the unit tests.

Reply:
Not really, it doesn't require that that large input. I will update it.